README.md

```diff
@@ -470,7 +470,7 @@ import { validate } from 'class-validator';
 
 class MyClass {
 	@MinLength(32, {
-		message: "EIC code must be at least 32 charatcers",
+		message: "EIC code must be at least 32 characters",
```

Comment:
This is already fixed in actual code base. Please do not edit lines out of scope of this PR.

Reply:
Please revert this.

Reply:
@rubiin